util/nrt_diff/nrtDiff.py

```diff
@@ -1265,17 +1265,17 @@ def XMLCloseEnough (path_left, path_right):
 def DTWDistance(left, right) :
     n = len(left)
     m = len(right)
-    DTW = [[0 for j in range(m+1)] for i in range(n+1)]
-
-    for i in range(1, n+1):
-        for j in range (1, m+1):
-            DTW[i][j] = 999999
+    DTW = [[999999 for j in range(m+1)] for i in range(n+1)]
```

Comment:
We don't put 0 in first line and first column, only the first case. is it normal ?

Reply:
yes, you are right, by chance it do not impacted a lot, it is now fixed